tests/test_dsp_pad_zeros.py

```diff
@@ -0,0 +1,48 @@
+import pyfar
```

Comment:
What's the reason for not putting the tests into test_dsp.py?

Reply:
not cluttering `test_dsp.py`. it's starting to get very crowded